app/views/application/_card.html.haml

```diff
@@ -13,4 +13,4 @@
           %strong.emojify.p-name= display_name(account, custom_emojify: true)
         %span
           = acct(account)
-          = fa_icon('lock') if account.locked?
+          = fa_icon('lock', { :data => ({hidden: true} unless account.locked?)})
```

Comment:
Why use data attribute instead of `style="display: none"`?

Reply:
No inline CSS allowed